app/src/test/java/nodomain/freeyourgadget/gadgetbridge/test/LanguageUtilsTest.java

```diff
@@ -57,11 +57,12 @@ public void testStringTransliterateArabic() {
         assertEquals("Farsi transiteration failed", farsiExpected, farsiActual);
     }
 
+    @Test
     public void testStringTransliterateBengali() throws Exception {
         // input with cyrillic and diacritic letters
         String[] inputs = { "অনিরুদ্ধ", "বিজ্ঞানযাত্রা চলছে চলবে।", "আমি সব দেখেশুনে ক্ষেপে গিয়ে করি বাঙলায় চিৎকার!" };
```

Comment:
Can you also add a some input texts that contains both Bengali and Latin characters? While I suppose that this would be very unusual, I would like to ensure that Gadgetbridge doesn't crash if someone sends such a message.

Reply:
Added a test string with Latin character in it. Working fine.